todo_app/data/mongo_items.py

```diff
@@ -0,0 +1,57 @@
+import os
+from bson import ObjectId
+import pymongo
+
+from todo_app.item import Item
+
+def get_collection():
+
+  client = pymongo.MongoClient(os.getenv("MONGODB_CONNECTION_STRING"))
+
+  db = client[os.getenv("MONGODB_DB_NAME")]
+
+  collection = db[os.getenv("MONGODB_COLLECTION_NAME")]
+
+  return collection
```

Comment:
Nothing major, but it's worth noting that your app is fairly slow currently - this is primarily because every interactions sets up a fresh connection with the DB. This is no terrible thing here, but if you're curious how we might improve that, a simple fix would be a basic variant of the singleton pattern:

```suggestion
_collection = None

def get_collection():
  global _collection
  if _collection is not None:
      return _collection

  client = pymongo.MongoClient(os.getenv("MONGODB_CONNECTION_STRING"))

  db = client[os.getenv("MONGODB_DB_NAME")]

  collection = db[os.getenv("MONGODB_COLLECTION_NAME")]
  _collection = collection

  return collection
```
(Or if we wanted a more OOP approach, wrap these methods in a class, and set the collection in the constructor!)

Reply:
Makes sense, I've implemented this, thanks!